CentralityiOS/CentralityiOS/ViewControllers/ModifyTaskModalViewController.m

```diff
@@ -39,9 +35,30 @@ - (void)viewDidLoad {
     [self.taskTitleInput addTarget:self action:@selector(textFieldDidChange:) forControlEvents:UIControlEventEditingChanged];
     self.taskTitleInput.delegate = self;
     self.taskDescInput.delegate = self;
-    IQKeyboardManager.sharedManager.enable = true;
+    IQKeyboardManager.sharedManager.enable = YES;
+    [self.shareButton setTitle:[self updateShareDisplayMessage] forState:UIControlStateNormal];
+    
+    if (!self.taskReadOnlyUsers){
+        self.taskReadOnlyUsers = [[NSMutableArray alloc] init];
+    }
+    if (!self.taskReadAndWriteUsers){
+        self.taskReadAndWriteUsers = [[NSMutableArray alloc] init];
+    }
+    if (!self.taskAcceptedUsers){
+        self.taskAcceptedUsers = [[NSMutableArray alloc] init];
+    }
 }
 
+- (NSString*)updateShareDisplayMessage{
+    NSString* shareDisplayMessage = [[NSString alloc]init];
+    if (self.taskSharedOwners.count == 1){
+        shareDisplayMessage = @"Sharing w/ 1 user";
```

Comment:
I'd either make this message _more_ unique (`@"Sharing with one user"`) or wrap it into the `shareDisplayMessage` below with a ternary like `self.taskSharedOwners.count != 1 ? @"s" : @""`